frontend/.env.example

```diff
@@ -0,0 +1 @@
+PUBLIC_BACKEND_ENDPOINT = "the url and port at which your backend is running, this can point to a deployed server if you only want to run the frontend"
```

Comment:
In dev mode, this file is used. In prod, this should be the same as `process.env`.